pkg/boot/linux.go

```diff
@@ -127,7 +127,7 @@ type named interface {
 	Name() string
 }
 
-func stringer(mod io.ReaderAt) string {
+func stringer(mod interface{}) string {
```

Comment:
change this back to io.ReaderAt

Reply:
any reason you prefer it to be `io.ReaderAt` ? -- I changed it b/c I needed it for a `io.Reader` equivalent. But now, not going with `io.Reader` path mentioned, so don't need it anymore. just curious. I can see  it good be specific

Reply:
regardless, done!

Reply:
actually, I do need it, in `pkg/boot/initrd.go`,

`names = append(names, stringer(initrd))`, where `initrd` is a io.Reader.